packages/arb-validator/rollupmanager/manager.go

```diff
@@ -145,7 +145,7 @@ func CreateManagerAdvanced(
 			}
 			man.Unlock()
 
-			currentProcessedBlockId := man.activeChain.CurrentBlockId()
+			currentProcessedBlockId := man.activeChain.CurrentEventId()
```

Comment:
Suggest renaming this variable, since it appears to no longer be a blockId

Reply:
Fixed